vllm/v1/worker/gpu_worker.py

```diff
@@ -223,9 +223,10 @@ def compile_or_warm_up_model(self) -> None:
         if get_pp_group().is_last_rank:
             max_num_reqs = min(self.scheduler_config.max_num_seqs,
                                self.scheduler_config.max_num_batched_tokens)
-            self.model_runner._dummy_sampler_run(
-                hidden_states=self.model_runner._dummy_run(
-                    num_tokens=max_num_reqs))
+            hidden_states = self.model_runner._dummy_run(num_tokens=max_num_reqs)
+            if hidden_states is not None:
+                self.model_runner._dummy_sampler_run(
+                    hidden_states=hidden_states)
```

Comment:
Is there any reason why minimax needs to return None for the hidden states in some cases?

It will be simpler if the gpu_worker can assume the `hidden_states` won't be none here, and I don't see why minimax would be a special case for this.